cmd/cosign/cli/attach.go

```diff
@@ -84,12 +84,16 @@ func attachAttestation() *cobra.Command {
 	o := &options.AttachAttestationOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "attestation",
-		Short:   "Attach attestation to the supplied container image",
-		Example: "  cosign attach attestation <image uri>",
-		Args:    cobra.ExactArgs(1),
+		Use:   "attestation",
+		Short: "Attach attestation to the supplied container image",
+		Example: `  cosign attach attestation --attestation <payload path> <image uri>
+
+  # attach multiple attestations to a container image
```

Comment:
suggestion: 

"attach attestations from multiple files to a container image"

Reply:
thanks